Docs/Configuration.tex

```diff
@@ -1875,18 +1875,21 @@ \subsection{Properties}\label{kernelprops}
   \textbf{Description}: Load selected kernel extensions (kexts) from the \texttt{OC/Kexts} directory.
 
   To be filled with \texttt{plist\ dict} values, describing each kext. Refer to
-  the \hyperref[kernelpropsadd]{Add Properties} section below.
+  the \hyperref[kernelpropsadd]{Add Properties} section below for details.
 
   \emph{Note 1}: The load order is based on the order in which the kexts appear in
   the array. Hence, dependencies must appear before kexts that depend on them.
 
   \emph{Note 2}: To track the dependency order, inspect the \texttt{OSBundleLibraries}
   key in the \texttt{Info.plist} file of the kext being added. Any kext included
-  in the \texttt{OSBundleLibraries} key is a dependency that must precede
-  the kext being added.
+  under the key is a dependency that must appear before the kext being added.
 
-  \emph{Note 3}: Kexts may have inner kexts (\texttt{Plugins}) included
-  in the bundle and each such plugin must be added separately.
+  \emph{Note 3}: Kexts may have inner kexts (\texttt{Plugins}) included in the bundle.
+  Such plugins must be added separately.
+
+  \emph{Note 4}: Some inner kexts may in fact be dependencies required by the host kext and
```

Comment:
Is such level of detail really needed?

Reply:
At least no body reading it will ever have any doubt or make any mistake. 
Can't be a bad thing don't you think?

Reply:
In my opinion, it takes the wrong approach really, what we have here is a total order over the OSBundleLibraries. Plugin kexts are simply not an exception.

Reply:
Lost now.

Better to close and leave as it was?

Reply:
Kexts may have inner kexts (\texttt{Plugins}) included in the bundle. Such plugins must be added separately. \texttt{Plugin} kext follow the same global ordering rules as all other kexts.